Lib/platform.py

```diff
@@ -612,6 +612,11 @@ def system_alias(system, release, version):
 
 ### Various internal helpers
 
+# Table for cleaning up characters in filenames.
+_SIMPLE_SUBSTITUTIONS = (
+    dict.fromkeys(map(ord, '/\\:;"()'), '-') | {ord(' '): '_'}
+)
```

Comment:
`str.translate()` is faster when the table is created by `str.maketrans()`.

```suggestion
_SIMPLE_SUBSTITUTIONS = str.maketrans(r' /\:;"()', '_-------')
```

Reply:
I totally forgot about maketrans(). Thanks!